etcdserver/api/rafthttp/stream.go

```diff
@@ -57,6 +57,8 @@ var (
 		"3.1.0": {streamTypeMsgAppV2, streamTypeMessage},
 		"3.2.0": {streamTypeMsgAppV2, streamTypeMessage},
 		"3.3.0": {streamTypeMsgAppV2, streamTypeMessage},
+		"3.4.0": {streamTypeMsgAppV2, streamTypeMessage},
+		"3.5.0": {streamTypeMsgAppV2, streamTypeMessage},
```

Comment:
Let's write a test that fails if there is not entry in this map for the current etcd major version on master.

Reply:
That way it's impossible to forget to update this and have a passing build.

Reply:
SGTM